schema.sql

```diff
@@ -1,5 +1,12 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+create database vet_clinic
+
+create table animals (
+id int PRIMARY KEY,
+name varchar(50),
+date_of_birth date,
+escape_attempts int,
+neutered boolean,
+weight_kg decimal
+);
```

Comment:
- [ ] [OPTIONAL] good job :+1: , Just one suggestion you can make the ID column NOT NULL and AUTO_INCREMENT: this means that the id value will be generated automatically by the database system for each new row added to the table.

Reply:
Ok, Will do That. Thanks. 😄